packages/fx-core/resource/package.nls.json

```diff
@@ -287,6 +287,11 @@
   "core.createProjectQuestion.projectType.outlookAddin.detail": "Customize the ribbon and Task Pane with your web content",
   "core.createProjectQuestion.projectType.outlookAddin.label": "Outlook Add-in",
   "core.createProjectQuestion.projectType.outlookAddin.title": "App Features Using an Outlook Add-in",
+  "core.createProjectQuestion.projectType.officeAddin.detail": "Extend Office applications and interact with content in Office documents and Outlook mails",
```

Comment:
Proposed: Extend Office apps to interact with content in Office documents and Outlook mails

Reply:
Thanks！I will change it.